UI/data/locale/en-US.ini

```diff
@@ -424,6 +424,9 @@ Basic.StatusBar.Delay="Delay (%1 sec)"
 Basic.StatusBar.DelayStartingIn="Delay (starting in %1 sec)"
 Basic.StatusBar.DelayStoppingIn="Delay (stopping in %1 sec)"
 Basic.StatusBar.DelayStartingStoppingIn="Delay (stopping in %1 sec, starting in %2 sec)"
+Basic.StatusBar.DynamicBitrate.Up="Network congestion clearing.\nIncreasing bitrate."
+Basic.StatusBar.DynamicBitrate.Stationary="Network congestion still ongoing.\nBitrate stationary."
```

Comment:
If this is the first time the user checks the tooltip, the "still" might throw them off. What about `Network congestion ongoing.\nMaintaining reduced bitrate.`?

Reply:
will do